packages/terra-search-field/src/SearchField.jsx

```diff
@@ -22,6 +23,22 @@ const propTypes = {
    */
   disableAutoSearch: PropTypes.bool,
 
+  /**
+   * Callback ref to pass into the inner input component.
+   */
+  inputRefCallback: PropTypes.func,
+
+  /**
+   * Custom input attributes to apply to the input field such as aria-label.
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  inputAttributes: PropTypes.object,
+
+  /**
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the `@private` syntax so this doesn't show up in the generated prop-types documentation.